README.md

```diff
@@ -78,6 +84,7 @@ Click in the table to download the most recent builds in the chosen language:
             <td align="center"><a href="https://github.com/Heegu-sama/Homm3BG/releases/download/v1.2/Heroes3_Ukrainska_Rules_Rewrite_12_Printable.pdf">завантажити</a></td>
             <td align="center"><a href="https://raw.githubusercontent.com/qwrtln/Homm3BG-build-artifacts/ua/main_ua.pdf">️завантажити</a></td>
             <td align="center"><a href="https://raw.githubusercontent.com/qwrtln/Homm3BG-build-artifacts/ua/printable_ua.pdf">завантажити</a></td>
+            <td align="center"><a href="https://raw.githubusercontent.com/qwrtln/Homm3BG-build-artifacts/en/economy_printable_ua.pdf">завантажити</a></td>
```

Comment:
en/ua?

Same in other rows here

Reply:
Thanks. Fixed

Maybe we don't need to have languge in path and the name but let's leave it like this for now. This PR is large enough :)

Reply:
We need them, because they are the git branch that the files are pushed to in another repo. If you have 7 workflows force pushing 3 blobs at the same time, you need 7 different branches.